source/constructs/api/catalog/service.py

```diff
@@ -819,6 +825,61 @@
     return result_list
 
 
+def get_table_property(table_id: str):
+    catalog_table = crud.get_catalog_table_level_classification_by_id(table_id)
+    if not catalog_table:
+        return {}
+    result_list = []
+    try:
+        client = get_boto3_client(catalog_table.account_id, catalog_table.region, catalog_table.database_type)
+        labels_str = ""
+        if catalog_table.label_ids:
+            labels = get_labels_by_id_list(catalog_table.label_ids)
+            if labels is not None:
+                labels_str = [{"id": label.id, "label_name": label.label_name} for label in labels]
+        result_list.append(["ResourceTags", labels_str])
+        result_list.append(["LastUpdated", catalog_table.detected_time])
+        if catalog_table.database_type == DatabaseType.S3.value:
+            result_list.append(["Account", catalog_table.account_id])
+            result_list.append(["Region", catalog_table.region])
+            result_list.append(["Objects", catalog_table.object_count])
+            result_list.append(["Size", catalog_table.size_key])
+            result_list.append(["Rows", catalog_table.row_count])
+            result_list.append(["S3Location", catalog_table.storage_location])
+            result_list.append(["Tags", __get_s3_tagging(catalog_table.database_name, client)])
+        elif catalog_table.database_type == DatabaseType.RDS.value or catalog_table.database_type == DatabaseType.GLUE.value:
+            response = client.describe_db_instances(DBInstanceIdentifier=catalog_table.database_name)
+            if "DBInstances" in response and len(response["DBInstances"]) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)